CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+# v2.3.5 (2024-08-21)
+
+## Orchestrator Changes
+
+### Kubernetes
+
+ * kubernetes (all): handle null in credential provider template ([#101])
```

Comment:
We can either list the versions explicitly or leave the implication it applies to all. 
```suggestion
 * Fix issue where a null value would fail to render the credential provider template for Kubernetes ([#101])
```